driver-core/src/main/com/mongodb/client/model/changestream/UpdateDescription.java

```diff
@@ -122,6 +142,35 @@ public List<TruncatedArray> getTruncatedArrays() {
         return truncatedArrays;
     }
 
+    /**
```

Comment:
Essentially copied from the spec but with Markdown converted by hand to HTML/Javadoc format